### PR TITLE
Remove unneeded dependency

### PR DESCRIPTION
**Pull request by davidfowl**
You don't need Microsoft.AspNetCore.Hosting.WindowsServices

### CODE REVIEW

File: IHostedServiceAsAService/IHostedServiceAsAService.csproj [hunk 1]
```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Hosting.WindowsServices" Version="2.1.1" />
+    <PackageReference Include="System.ServiceProcess.ServiceController" Version="4.5.0" />
     <PackageReference Include="Microsoft.Extensions.Hosting" Version="2.1.1" />
```
Comment (stevejgordon) [inline IHostedServiceAsAService/IHostedServiceAsAService.csproj:12]: Just realised that I can reduce to just 

```
<ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Hosting.WindowsServices" Version="2.1.1" />
  </ItemGroup>
```
which then brings in both Microsoft.Extensions.Hosting and System.ServiceProcess.ServiceController. Is there a benefit to pulling both in directly?

**Review by stevejgordon — COMMENTED** [review]
Thanks @davidfowl. Left a question for you as now a little confused on the best option,

File: IHostedServiceAsAService/IHostedServiceAsAService.csproj (hunk 1 above)
Comment (stevejgordon) [reply IHostedServiceAsAService/IHostedServiceAsAService.csproj:12]: Actually seems then I can't resolve HostBuilder, so potentially scratch this query while I check the package graph again!

Comment (stevejgordon) [reply IHostedServiceAsAService/IHostedServiceAsAService.csproj:12]: Okay, yeah I see, brings in only Microsoft.Extensions.Hosting.Abstractions under Microsoft.AspNetCore.Hosting.

Comment (ehsansajjad465) [reply IHostedServiceAsAService/IHostedServiceAsAService.csproj:12]: So what should be added as nuget, **Microsoft.AspNetCore.Hosting.WindowsServices** or the other two packages ?

**Pull request merged**